pom.xml

```diff
@@ -68,30 +68,32 @@
         <lombok.version>1.18.36</lombok.version>
         <lombok-maven-plugin.version>1.18.20.0</lombok-maven-plugin.version>
         <hadoop.version>3.4.1</hadoop.version>
-        <hudi.version>0.14.0</hudi.version>
+        <hudi.version>1.1.0-SNAPSHOT</hudi.version>
         <aws.version>2.29.40</aws.version>
-        <hive.version>2.3.9</hive.version>
+        <databricks.version>0.41.0</databricks.version>
+        <aws.httpclient.version>4.5.13</aws.httpclient.version>
+        <hive.version>3.1.3</hive.version>
         <maven-source-plugin.version>3.3.1</maven-source-plugin.version>
         <maven-javadoc-plugin.version>3.8.0</maven-javadoc-plugin.version>
         <maven-gpg-plugin.version>3.2.4</maven-gpg-plugin.version>
         <maven-deploy-plugin.version>3.1.1</maven-deploy-plugin.version>
         <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
         <mockito.version>5.15.2</mockito.version>
-        <parquet.version>1.15.1</parquet.version>
+        <parquet.version>1.13.1</parquet.version>
         <protobuf.version>3.25.5</protobuf.version>
         <scala12.version>2.12.20</scala12.version>
         <scala13.version>2.13.15</scala13.version>
         <scala.version>${scala12.version}</scala.version>
         <scala.binary.version>2.12</scala.binary.version>
-        <spark.version>3.4.2</spark.version>
-        <spark.version.prefix>3.4</spark.version.prefix>
-        <iceberg.version>1.4.2</iceberg.version>
-        <delta.version>2.4.0</delta.version>
+        <spark.version>3.5.2</spark.version>
+        <spark.version.prefix>3.5</spark.version.prefix>
+        <iceberg.version>1.5.2</iceberg.version>
+        <delta.version>3.0.0</delta.version>
```

Comment:
Can we pull the spark, iceberg, and delta version upgrades into their own PR? It would help shrink the size of this PR